ui/src/components/AddServerDialog.tsx

```diff
@@ -526,16 +564,19 @@ export function AddServerDialog({ open, onOpenChange, onAddServer, onError }: Ad
               />
             </div>
 
-            <Tabs defaultValue="command" value={activeTab} onValueChange={(v) => setActiveTab(v as "command" | "url")}>
+            <Tabs defaultValue="url" value={activeTab} onValueChange={(v) => setActiveTab(v as "command" | "url")}>
               <TabsList className="grid w-full grid-cols-2">
-                <TabsTrigger value="command" className="flex items-center gap-2">
-                  <Terminal className="h-4 w-4" />
-                  Command
-                </TabsTrigger>
                 <TabsTrigger value="url" className="flex items-center gap-2">
                   <Globe className="h-4 w-4" />
                   URL
                 </TabsTrigger>
+                {renderTabTrigger(
+                  "command",
```

Comment:
Left the tabs as "url" and "command" for now although I can't help but feel like this dialog/the tabs need a bit of an overhaul given that ultimately, [`url` = `RemoteMCPServer`](https://github.com/kagent-dev/kagent/blob/bdbde1109e7fd214a2536940043a6afef8e07c92/ui/src/components/AddServerDialog.tsx#L347-L393) and [`command` = `MCPServer`](https://github.com/kagent-dev/kagent/blob/bdbde1109e7fd214a2536940043a6afef8e07c92/ui/src/components/AddServerDialog.tsx#L283-L345) (which incidentally now needs to support a far broader range of `cmd` than just `npx` and `uvx`).

Reply:
I agree. As a start, it might make sense to rename them `RemoteMCPServer` and `MCPServer`.

Reply:
Do you want to do that as part of this PR or as a separate one?

Reply:
up to you

Reply:
Let's deal with it separately since it will be needed regardless of what happens with this PR.